src/app/components/mnoe-product-markups-list/mnoe-product-markups-list.html

```diff
@@ -0,0 +1,63 @@
+<mno-widget icon="fa-percent" heading="{{$ctrl.markups.widgetTitle | translate}}" is-loading="markups.loading">
+  <mno-widget-header>
+    <!--{{ 'mnoe_admin_panel.dashboard.product_markups.widget.list.title' | translate}}-->
+  </mno-widget-header>
+  <mno-widget-body class="large no-padding">
+    <div>
+      <table class="table" st-pipe="$ctrl.callServer" st-table="$ctrl.displayed">
+        <table st-pipe="$ctrl.callServer" st-table="$ctrl.listOfProductMarkups" class="table table-striped" >
+          <thead>
+          <tr>
+            <th st-sort="product.name" class="col-sm-4" translate>mnoe_admin_panel.dashboard.product_markups.widget.list.table.product</th>
+            <th st-sort="organization.name" class="col-sm-4" translate>mnoe_admin_panel.dashboard.product_markups.widget.list.table.customer</th>
+            <th st-sort="percentage" class="col-sm-2 text-center" translate>mnoe_admin_panel.dashboard.product_markups.widget.list.table.percentage</th>
+            <th class="col-sm-1"></th>
+            <th class="col-sm-1"></th>
+          </tr>
+          <tr>
+            <th>
+              <input st-search="product.name" placeholder="{{'mnoe_admin_panel.dashboard.product_markups.widget.list.placeholder.search_product' | translate}}" class="form-control input-sm search-bar col-sm-4" type="text"/>
+            </th>
+            <th>
+              <input st-search="organization.name" placeholder="{{'mnoe_admin_panel.dashboard.product_markups.widget.list.placeholder.search_customer' | translate}}" class="form-control input-sm search-bar col-sm-4" type="text"/>
+            </th>
+            <th></th>
+            <th></th>
+            <th></th>
+          </tr>
+          </thead>
+          <tbody ng-show="$ctrl.markups.loading">
+          <tr>
+            <td colspan="3" class="text-center" translate>mnoe_admin_panel.dashboard.product_markups.widget.list.loading_product_markups</td>
+          </tr>
+          </tbody>
+          <tbody ng-show="!$ctrl.markups.loading" >
+          <tr ng-repeat="pmElem in $ctrl.listOfProductMarkups">
+            <td class="col-sm-4">{{pmElem.product.name}}</td>
```

Comment:
The `td` elements take the same width than their corresponding `th`. So you can remove the `col-sm-*` on the `td`s.